target/linux/ramips/dts/mt7621_asus_rt-n56u-b1.dts

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "asus,rt-n56u-b1", "mediatek,mt7621-soc";
+	model = "ASUS RT-N57U B1";
```

Comment:
These are different. Copy/paste error?

Reply:
Yes. It is error. RT-AC57U is the similar hardware, different gpio, port number location, SoC here 1core.

Reply:
RT-N56U B1 -true value